nixos/modules/services/networking/radicale.nix

```diff
@@ -84,9 +150,25 @@ in
         ));
         User = "radicale";
         Group = "radicale";
+        StateDirectory = "radicale/collections";
+        UMask = 0027;
+        PrivateTmp = true;
+        ProtectSystem = "strict";
+        ProtectHome = true;
+        PrivateDevices = true;
+        ProtectKernelTunables = true;
+        ProtectKernelModules = true;
+        ProtectControlGroups = true;
+        NoNewPrivileges = true;
+        ReadWritePaths = if
+          hasAttrByPath [ "storage" "filesystem_folder" ] cfg.settings
+        then
+          [ cfg.settings.storage.filesystem_folder ]
+        else
+          [ "/var/lib/radicale/collections" ];
```

Comment:
If we don't add an option to add a path to this list, we should use `mkDefault` so people can still set `systemd.services.radicale.serviceConfig.ReadWritePaths` themselves.

Reply:
Oh, true.